include/rcutils/filesystem.h

```diff
@@ -105,16 +105,20 @@ RCUTILS_PUBLIC
 bool
 rcutils_is_readable_and_writable(const char * abs_path);
 
-/// Concatenate path adding the right delimiter according to the platform.
+/// Return a newly allocated string that contains left_hand_path, followed by
+/// the correct delimiter for the platform, followed by right_hand_path.
```

Comment:
Brief description should fit on a single line. The extended description should go in the docblock below

Reply:
Thanks, I've changed this to be a single line now.